pkg/stub/handler.go

```diff
@@ -88,16 +95,77 @@ type Handler struct {
 
 	skippedTemplates    map[string]bool
 	skippedImagestreams map[string]bool
+
+	defaultResourceTimer *time.Timer
 }
 
-func (h *Handler) manageDockerCfgSecret(deleted bool, operatorCfg *v1alpha1.SamplesResource, secret *corev1.Secret) error {
-	// clear out temporary secret cache
-	h.registrySecret = corev1.Secret{}
+func (h *Handler) GoodStatusUpdate(srcfg *v1alpha1.SamplesResource, newStatus corev1.ConditionStatus) error {
```

Comment:
how is this different from a "not good" status update?

why is it not used for the CredentialsExist status?

Reply:
the `processError` functionality is the "not good" status update  

I'm certainly open to another name

and yes, I could leverage this for the CredentialsExists status